### PR TITLE
unify folder structure

### PR DESCRIPTION
**Pull request by denesnori**
Esraa's and my branch followed a different folder structure. I just unified the folder structure.

### CODE REVIEW

**Review by njsfield — CHANGES_REQUESTED** [review]
Would it be okay to add a .gitignore file, and inside put .DS_Store. Then delete the .DS_Stores and commit again?

**Comment by denesnori** [comment]
It's done, I've added the .gitignore.

**Review by njsfield — APPROVED** [review]

**Comment by denesnori** [comment]
I added the navbar to all the pages, and now all the links work. :)

**Review by jbesraa — APPROVED** [review]

File: js/nextQuestion.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+function gotToNextQuestion(id){
+  id++;
+  let url=window.location.href;
+  let base=url.split(/\d/)[0]
+  console.log(url);
```
Comment (des-des) [inline js/nextQuestion.js]: ops

File: 3cards_layout.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            <div class="card-content">
+              <p>{{getMedication questions}}</p>
+            </div>
+            <div class="card-action">
+              <a href="#">This is a link</a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- We have an overcomplecated data structure for the questions and treatment options.
```
Comment (des-des) [inline 3cards_layout.html]: Can these comments move into js file?

**Comment by des-des** [comment]
@denesnori it is not really helpful to push code unrelated to my review, as I cannot review it independently of what i have already reviewed.

**Comment by denesnori** [comment]
Sorry, I"ll pay attention to it the next time, I forgot to ckeck out to a new branch when I finished what you asked for.

**Review by des-des — APPROVED** [review]

File: js/previousQuestion.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+function goToPreviousQuestion(id){
+  id--;
+  console.log(id);
```
Comment (des-des) [inline js/previousQuestion.js]: log

**Review by des-des — CHANGES_REQUESTED** [review]

File: js/cards.js [hunk 4]
```diff
@@ -109,8 +109,12 @@ $(document).ready(function(){
         }
 
     }});
-
-
+    let url=window.location.href;
+    let newUrl= url.split("/")
+    let key=newUrl.pop();
+    console.log(key);
```
Comment (des-des) [inline js/cards.js]: :fire:

File: js/cards.js [hunk 5]
```diff
@@ -109,8 +109,12 @@ $(document).ready(function(){
...
+    let num=key.split("=")[1]||1;
+    console.log(num);
```
Comment (des-des) [inline js/cards.js]: :fire:

**Review by des-des — CHANGES_REQUESTED** [review]

**Review by des-des — APPROVED** [review]

**Comment by des-des** [comment]
kk ignore all the reviews, was going though each commit

**Comment by denesnori** [comment]
thanks :) I only removed the console logs in one of the last commits when everything was working. Sorry for the trouble.

**Comment by des-des** [comment]
thats cool!

**Pull request merged**